docs/Development.md

```diff
@@ -24,3 +24,15 @@ npm run watch
 
 Open [`http://127.0.0.1:8080/demo`](http://127.0.0.1:8080/demo) to start playing!
 Any changes you make to source files will be visible when you refresh the browser page.
+
+## Updating Versions
+
+Our CircleCI builds will publish new versions as listed in `package.json`.
+To update a version:
+
+1. Create a new branch locally
+2. Run [`npm version`](https://docs.npmjs.com/cli/version) to update `package.json` and `package-lock.json`
```

Comment:
Does `npm version` have to be followed by a number? or is that pre-set?

Reply:
I've always used the semver spec types, like `npm version patch` or `npm version minor`...

Reply:
Oh is there a guideline for which one to use and in which cases? 

I stumbled on this: https://raw.githubusercontent.com/wiki/vrjuggler/vrjuggler/VersionBumping.md but is there some reference that you were following?

Reply:
Oh gotcha! https://semver.org/ is the base guide, but since it's kind of verbose, here's the tl;dr I try to run with:

* *Major*: If there's a "breaking" change in the API. Like, if someone using the package would either experience build breaks or drastically different behavior without changing how they use the package 
* *Minor*: Feature additions that aren't breaking changes
* *Patch*: Bug fixes that aren't breaking changes

...there's intentionally a bit of ambiguity there, since some people accidentally might end up relying on bugs or lacks of features... so semver is a bit of a lie, sometimes. But it's a good rule of thumb.

Reply:
I see, yea, let's add it and then we can clarify or amend if need be :)